# /costmap_updates反映待ちカウンタ
# - /costmap_updates受信時に1インクリメントされ、カウンタ数に到達した際に
# - ローカルに保持するコストマップに反映する
costmap_updates_wait_count: 5
# プログラム実行間隔（hz）
program_cycle_frequency: 10
# ソシオ地図配信カウンタ
# - プログラムループ毎に1インクリメントされ、
# - カウンタ数に到達した際に地図を配信する
publish_cycle_count: 10
# コスト表示カウンタ
# - ソシオ地図配信間隔毎に1デクリメントされ、0に到達した際に
# - ローカルに保持するソシオ地図からコストを削除する
cost_output_count: 10
# コスト存在時間チェックカウンタ
# - ソシオ地図配信間隔毎にコストが存在する場合に1デクリメントされ、
# - 0に到達した際にローカルに保持するソシオ地図にコストを反映する
cost_existence_check_count: 2

